modules/evm/src/lib.rs

```diff
@@ -1036,14 +1036,8 @@ pub mod module {
 		#[transactional]
 		pub fn deploy(origin: OriginFor<T>, contract: EvmAddress) -> DispatchResultWithPostInfo {
```

Comment:
also rename this to publish_contract to align with the predeploy contract

Reply:
:+1: Note Changing the name will break some tests in bohdi.js and any thing else that uses `tx.evm.deploy()` downstream

Reply:
yeah that's fine. better to break it before we release it